mathics/builtin/atomic/numbers.py

```diff
@@ -5,9 +5,9 @@
 # printed output, so be carful not to add then mid-sentence.
 
 """
-Numeric Evaluation and Precision
```

Comment:
In mathics.builtin.numeric we had things like `NIntegrate`. Should them be here now?

Reply:
I would like to follow the Mathematica docs organization. However this was taking too much time and I wanted to finish getting a start here. 

Feel free to adjust this along the organization of the Mathematica docs. 

What I wanted really to make more headway this weekend towards trying to just come up with something that we could start to discuss and document the current evaluation process.  (And of course this is a prerequisite for discussing the current design and what proposed changes we might want to make. 

But my take is that in order to make forward progress spending a little bit of time trying to organize what we have will be helpful for everyone to be on the same page.